<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="screen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>144</height>
     </rect>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionRegisters"/>
    <addaction name="actionMemory"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionExit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionRegisters">
   <property name="text">
    <string>Registres</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="text">
    <string>MÃ©moire</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRegisters</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDebugRegistersWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMemory</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openDebugMemoryWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectROMFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openDebugRegistersWindow()</slot>
  <slot>openDebugMemoryWindow()</slot>
  <slot>selectROMFileName()</slot>
 </slots>
</ui>
